demos/defender/defender_demo/metrics_collector.h

```diff
@@ -0,0 +1,145 @@
+/*
+ * AWS IoT Device SDK for Embedded C V202009.00
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#ifndef METRICS_COLLECTOR_H_
+#define METRICS_COLLECTOR_H_
+
+/* Standard includes. */
+#include <stdint.h>
+
+/**
+ * @brief Return codes from metrics collector APIs.
+ */
+typedef enum
+{
+    MetricsCollectorSuccess = 0,
+    MetricsCollectorBadParameter,
+    MetricsCollectorFileOpenFailed,
+    MetricsCollectorParsingFailed
+} MetricsCollectorStatus_t;
+
+/**
+ * @brief Represents network stats.
+ */
+typedef struct NetworkStats
+{
+    uint32_t bytesReceived;
```

Comment:
What is the difference between "_bytes_" and "_packets_" for the network stats?
Documentation about them would help the reader understand the difference between them